crates/cli-support/src/js/binding.rs

```diff
@@ -550,11 +550,13 @@ fn instruction(js: &mut JsBuilder, instr: &Instruction, log_error: &mut bool) ->
         | Instruction::CallTableElement(_)
         | Instruction::DeferCallCore(_) => {
             let invoc = Invocation::from(instr, js.cx.module)?;
-            let (params, results) = invoc.params_results(js.cx);
+            let (mut params, results) = invoc.params_results(js.cx);
 
             let mut args = Vec::new();
             let tmp = js.tmp();
             if invoc.defer() {
+                // substract alignment
+                params -= 1;
```

Comment:
It's not ideal to have to hardcode this to only work with `free`. I might push a commit of my own to work around this.

Reply:
My idea didn't pan out, so I guess we can go with this. `DeferCallCore` should probably be renamed to `DeferFree` or something now that it only works with `free`, though.

Reply:
See https://github.com/rustwasm/wasm-bindgen/pull/3463/files#r1219300930.